scripts/main/view.js

```diff
@@ -216,9 +221,14 @@ view.album = {
 
 			if (photosData !== "") {
 				if (lychee.layout === 1) {
-					photosData = '<div class="justified-layout">' + photosData + "</div>";
+					// The CSS class 'relayouting' prevents the DIV from being
```

Comment:
With my "grammar nazi" hat on, I think strictly speaking the class should be called `relayingout`? Personally I think I would drop the `re` as well, i.e., `layingout`...

Reply:
I don't mind. You are the English pro. I will change it. But maybe "laying-out" with a hyphen for better readability?

Reply:
Fine with me...

Reply:
Fixed in https://github.com/LycheeOrg/Lychee-front/pull/334/commits/dcfc2c2dfeeebf03d64fd7c7526002327f3240cd.